src/main/java/apaintus/models/snapgrid/SnapGrid.java

```diff
@@ -5,56 +5,50 @@
 import java.util.List;
 
 public class SnapGrid {
-	private double spacing = 0;
+	private double size = 0;
```

Comment:
as mention in upper comments, we are not talking about size here and it will be changed in the release of the rulers features